components/ImprovedSearch/CustomClearRefinement.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button } from "@equinor/eds-core-react"
+import { tokens } from "@equinor/eds-tokens"
+import React from "react"
+import { useClearRefinements, UseClearRefinementsProps } from "react-instantsearch-hooks-web"
+import { useIntl } from "react-intl"
+import styled from "styled-components"
+
+const ButtonNew = styled(Button)`
+  color: ${tokens.colors.text.static_icons__default.hex};
+  background: ${tokens.colors.infographic.primary__moss_green_13.hex};
+  border-width: 0;
+  font-weight: 14pt;
+  padding-left: 8.15rem;
+  :hover {
```

Comment:
Did Espen tell you to override the hover effect from EDS? 🤔  I think it is strange that it does not have the hover effect. Should it not be a ghost button like the ones in the header instead?

Reply:
actually the color coming by default from hiver is not what we  wanted so i have to use it in styling and get the color of our choice

Reply:
OK, it still feels a little bit weird that we don't have a hover effect on this button but if that's what Espen wants 🤷‍♀️